packages/terra-clinical-onset-picker/src/OnsetPicker.scss

```diff
@@ -12,4 +12,26 @@
     margin-bottom: 0.357em;
     margin-right: 0.357em;
   }
+
+  .precision {
+    min-width: 150px;
+  }
+
+  .granularity {
+    min-width: 100px;
+  }
+
+  .month {
+    min-width: 150px;
+  }
+
+  .age-unit {
+    min-width: 100px;
+  }
+
+  @media screen and (min-width: 544px) {
```

Comment:
The terra-breakpoints package has media queries that you can import: https://github.com/cerner/terra-core/blob/master/packages/terra-breakpoints/src/_media-queries.scss#L1

Reply:
It seems the sass loader is having issues finding that file to import after I include the package. Is there a change in the webpack configuration that needs to happen first for that sass import to work properly?

Reply:
Try adding this to the top of the file `@import '~terra-breakpoints/lib/media-queries';`
If that doesn't work, let me know and we can debug.

Reply:
Ah that import seemed to have worked... might want to update the docs for that project!